handler.js

```diff
@@ -0,0 +1,72 @@
+import { client } from 'octonode'
+
+export async function checkSpecification(event, context, callback) {
+  let response
+  const githubClient = client(process.env.GITHUB_TOKEN)
+
+  const body = JSON.parse(event.body)
+  // when creating the webhook
+  if (body && ('hook_id' in body)) {
+    response = {
+      statusCode: 200,
+      body: `Hello ${body.sender.login}, the webhook is now enabled for ${body.repository.full_name}, enjoy!`,
+    }
+
+    if (!body.hook.events.includes('pull_request')) {
```

Comment:
`body.hook.events` always exists ?

Reply:
Since I check for `hook_id` few lines above, yep

Reply:
A little bit weird to doing that btw, what not checking `hook` existance rather that `hook_id` ?